pkg/datasources/resource_monitors.go

```diff
@@ -43,41 +39,41 @@ var resourceMonitorsSchema = map[string]*schema.Schema{
 
 func ResourceMonitors() *schema.Resource {
 	return &schema.Resource{
-		Read:   ReadResourceMonitors,
-		Schema: resourceMonitorsSchema,
+		ReadContext: ReadResourceMonitors,
+		Schema:      resourceMonitorsSchema,
+		Description: "Datasource used to get details of filtered resource monitors. Filtering is aligned with the current possibilities for [SHOW RESOURCE MONITORS](https://docs.snowflake.com/en/sql-reference/sql/show-resource-monitors) query (`like` is all supported). The results of SHOW is encapsulated in one output collection.",
```

Comment:
Such a description is missing from `database_roles`.

Reply:
Done